src/containers/GlossPage/components/TranscriptionField.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Input, FieldRemoveButton } from '@ndla/forms';
+import { spacing } from '@ndla/core';
+import { useTranslation } from 'react-i18next';
+import styled from '@emotion/styled';
+import FormikField from '../../../components/FormikField';
+
+interface Props {
+  label: string;
+  name: string;
+  value?: string;
+  removeField: () => void;
+}
+
+const StyledTranscriptionField = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  margin: ${spacing.small} 0;
+
+  > :first-child {
+    padding-right: ${spacing.small};
+  }
+
+  > :last-child {
+    margin-bottom: ${spacing.small};
+  }
```

Comment:
Emotion kommer til å gi oss konsoll-kjeft for dette. Er det en annen måte å style det på?